src/modules/new/index.ts

```diff
@@ -238,6 +237,8 @@ async function run(operation: Command, USAGE: CLI): Promise<any> {
                 USAGE
             );
 
+            // [2]d Optional module questions need to be asked after project name is requested.
+            modulesToInstall = await OPTIONAL_MODULES.requestPresetSelection();
```

Comment:
This works. A caveat though is that if the user cancels (Ctrl  + C) the project creation while selecting from presets, there would be files already created for the project (these would be on the user's device). Are we okay with that happening?

Don't think that should hinder us at the moment, but its something for us to think about.

Reply:
That's interesting, I'll create a ticket for it and also raise the team's awareness to the subject matter.